docker-compose.yml

```diff
@@ -1,4 +1,12 @@
-version: '3'
+version: '2.1'
```

Comment:
I'm totally fine with the version change. Just curious about the reason downgrading from `3` to `2.1`.

Reply:
Sorry it's a mistake. I'll fix it in the next PR.

Reply:
Fixed here https://github.com/waltzofpearls/healthstats-collector/pull/3